components/CollectiveNavbar.js

```diff
@@ -441,108 +444,113 @@ const CollectiveNavbar = ({
       </InfosContainerV2>
       {/** Main navbar items */}
       {/* TO DO: they don't shrink properly when the container gets small - needs work */}
-      <Container
-        backgroundColor="#fff"
-        zIndex={1}
-        display={isExpanded ? 'flex' : ['none', 'flex']}
-        flexDirection={['column', 'row']}
-        flexBasis="600px"
-        flexShrink={2}
-        flexGrow={1}
-        justifyContent={['space-between', null, 'flex-start']}
-        minWidth={0}
-        order={[0, 3, 0]}
-        borderTop={['none', '1px solid #e1e1e1', 'none']}
-        overflowX="auto"
-      >
-        {isLoading ? (
-          <LoadingPlaceholder height={43} minWidth={150} mb={2} />
-        ) : (
-          <Fragment>
-            {sections.map(section => (
-              <MenuLinkContainerV2
-                mr={[0, 3]}
-                key={section}
-                isSelected={section === selected}
-                onClick={() => {
-                  if (isExpanded) {
-                    setExpanded(false);
-                  }
-                  if (onSectionClick) {
-                    onSectionClick(section);
+      {!onlyInfos && (
```

Comment:
@Betree this looks like a huge change but all that's being done is wrapping the second two Containers (for the main navbar items and the CTAs/admin panel in the conditional for `!onlyInfos`. Otherwise it's just the padding, margin, and `showBackButton` changes in this file, so hopefully not too  much of a merge conflict.